FV3/atmos_cubed_sphere/model/fv_arrays.F90

```diff
@@ -1025,6 +1025,8 @@ module fv_arrays_mod
    logical :: write_coarse_restart_files = .false.  ! Whether to write coarse restart files
    logical :: write_coarse_diagnostics = .false.  ! Whether to enable writing coarse diagnostics
    logical :: write_only_coarse_intermediate_restarts = .false.  ! Whether to write only coarse intermediate restart files (if do_coarse_graining is .true.)
+   logical :: restart_from_agrid_winds = .false.  ! Whether to restart from A-grid winds
+   logical :: optional_dgrid_vel_rst = .false.  ! Whether to write out optional D-grid winds when restart_from_agrid_winds is active
```

Comment:
I'd suggest `write_dgrid_vel_rst` for this second flag (it's unfortunate that the existing `agrid_vel_rst` doesn't reference writing). I see what you're getting at with the optional, but I think it's enough to explain in the comment. You could do `write_optional_dgrid_vel_rst` if you're feeling verbose.

Reply:
Indeed, having a verb in there is better.

Reply:
Done